phpstan-baseline.neon

```diff
@@ -171,7 +171,16 @@ parameters:
 			path: src/Utils.php
 
 		-
-			message: "#^Unreachable statement \\- code above always terminates\\.$#"
+			message: "#^Strict comparison using === between '' and non-empty-string will always evaluate to false\\.$#"
 			count: 1
-			path: src/Utils.php
+			path: src/UriResolver.php
+
+		-
+			message: "#^Method GuzzleHttp\\\\Psr7\\\\LazyOpenStream::__get\\(\\) throws checked exception InvalidArgumentException but it's missing from the PHPDoc @throws tag.$#"
+			count: 1
+			path: src/StreamDecoratorTrait.php
 
+		-
+			message: "#^Method GuzzleHttp\\\\Psr7\\\\MultipartStream::__get\\(\\) throws checked exception InvalidArgumentException but it's missing from the PHPDoc @throws tag.$#"
+			count: 1
+			path: src/StreamDecoratorTrait.php
```

Comment:
It was impossible to solve this one, it seems to be bug in phpstan, im already in touch with @ondrejmirtes about creating issue with bugreport